pkg/subctl/cmd/join.go

```diff
@@ -71,6 +73,7 @@ func addJoinFlags(cmd *cobra.Command) {
 	cmd.Flags().BoolVar(&ipsecDebug, "ipsec-debug", false, "Enable IPsec debugging (verbose logging)")
 	cmd.Flags().BoolVar(&submarinerDebug, "subm-debug", false, "Enable Submariner debugging (verbose logging)")
 	cmd.Flags().BoolVar(&noLabel, "no-label", false, "skip gateway labeling")
+	cmd.Flags().StringVar(&kubefedController,  "kubefed-controller", "" , "Kubefed Controller Context Name")
```

Comment:
Use same descrption as used by kubefedctl:
 `--host-cluster-context string   Host cluster context`
It isn't clear that it needs the name of the host.

Also, this might be better option:
`--host-cluster-name string      If set, overrides the use of host-cluster-context name in resource names created in the target cluster. This option must be used when the context name has characters invalid for kubernetes resources like "/" and ":".`

>

Reply:
Done.